drivers/adc/ad9081/ad9081.c

```diff
@@ -707,18 +707,37 @@ static int32_t ad9081_setup(struct ad9081_phy *phy)
 	mdelay(10);
 
 	if (phy->jesd_rx_clk) {
-		mdelay(10);
-		ret = clk_enable(phy->jesd_rx_clk);
-		if (ret < 0) {
+		int timeout = 2000;
```

Comment:
you can declare timeout at the beginning of the function and only initialize it here.
And also for consistency you should use int32_t instead of int.

Reply:
I prefer variables in the context where they are used but I think we have a rule about this so I'll move it at the top